src/components/pages/Authentication/hooks.ts

```diff
@@ -100,20 +102,18 @@ export default () => {
       if (res.status !== 200) {
         setNotification({
           type: "error",
-          text: intl.formatMessage({ defaultMessage: "Something went wrong. Please try again." }),
+          text: res.data.error,
```

Comment:
Is this error meaningful for the user? Most errors from the backend are not super meaningful and have zero punctuation (no periods and all lowercase). I know it is something eventually we want to update but for now unless we make a point to write user-friendly/user-facing error messages I feel like even a generic message like the one this change is replacing is better.
@rot1024 What do you think?

Reply:
@yk-eukarya , Yasser discuss this.

Reply:
Currently, you can either hide the error message at all, or you can just give the error message as it is.

Backends running in the production environment do not return error messages containing internal details. Error messages are written and human-readable, so it is acceptable to display them as they are. 

However, if this were to be made multilingual, additional implementation would be required. Since error messages are changed at the convenience of the backend, it would be better to include error message translations on the backend side, but so far no mechanism has been implemented for error messages based on the user's language preferences.

Reply:
How about returning an error code with the message, something like this:
```
{
   meesage: "error message in English only",
   code: "E102"
}
```
the message can help the developer to figure out what is the error. the code can help to implement the translation on the front-end side. in this scenario, the translation has nothing to do with the back end and we can keep all translation stuffs will be in the front end.
@rot1024 What do you think?

Reply:
That method is vulnerable to change in our project structure. Every time we add or change the error messages in the backend, we have to change the frontend. It would be better if the backend had a translation of the error messages and returned the translated results.